### PR TITLE
Make the analyzer able to guess cities based on station names in the tweet

### PR DESCRIPTION
**Pull request by progval**

### CODE REVIEW

**Comment by RaitoBezarius** [comment]
:+1:

File: backend/analyzer.py [hunk 1]
```diff
@@ -27,11 +34,184 @@ def from_friendly_name(name):
...
+Transportation = collections.namedtuple('Transportation', 'type line')
 
 
-line_number_pattern = re.compile(r'^([0-9]{1,3}|[a-z]{1,2})$', re.I)
 
-Transportation = collections.namedtuple('Transportation', 'type line')
+################################################
+# Normalization
+
+class shortcuts:
```
Comment (RaitoBezarius) [inline backend/analyzer.py:44]: Why a lowercase class name?

Comment (progval) [inline backend/analyzer.py:44]: it's a namespace

File: backend/analyzer.py [hunk 2]
```diff
@@ -27,11 +34,184 @@ def from_friendly_name(name):
...
+    """namespace"""
+    remove = lambda L, x: list(filter(lambda i:i not in x, L))
```
Comment (RaitoBezarius) [inline backend/analyzer.py:46]: You should add a space after `lambda i:` like you did with others lambda declarations, or is there a reason?

Comment (progval) [inline backend/analyzer.py:46]: Does it matter?

Comment (RaitoBezarius) [inline backend/analyzer.py:46]: Not really, just was wondering!

File: backend/analyzer.py (hunk 1 above)
Comment (RaitoBezarius) [inline backend/analyzer.py:44]: Understood!

**Comment by RaitoBezarius** [comment]
Excited to merge it, can I ?

**Comment by progval** [comment]
I thought your :+1: meant you wanted me to, so I already did

**Comment by RaitoBezarius** [comment]
Haha, no problem! :D

**Pull request merged**